openedx/features/course_experience/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Unified course experience settings
+
+UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
+UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
```

Comment:
That's a good idea. I was having trouble naming the new flag, due to the way I named the previous one. I'll add a comment, but I'm also happy to use a different name for the new one. We could also consider renaming the old one, but then when it goes live we'll have to switch the production flag. This may not be too bad given that we haven't switched it on yet (except for a few course team members).